### PR TITLE
Fix graph modules

### PR DESCRIPTION
**Pull request by hyeok9855**
- [x] I've read the [.github/CONTRIBUTING.md](.github/CONTRIBUTING.md) file
- [x] My code follows the typing guidelines
- [x] I've added appropriate tests
- [x] I've run pre-commit hooks locally

## Description
I found a counter-intuitive design in the calculation of logits for `exit_action` in the modules for graph states. Specifically, in `GraphEdgeActionGNN`,
```python
            action_type[..., GraphActionType.ADD_EDGE] = 1 - exit_action
            action_type[..., GraphActionType.EXIT] = exit_action
```
This doesn't make sense since exit_action is not a probability but a logit. I fixed this as follows:
```python
            action_type[..., GraphActionType.ADD_EDGE] = F.logsigmoid(-exit_action)
            action_type[..., GraphActionType.EXIT] = F.logsigmoid(exit_action)
```
Note that `F.logsigmoid(-exit_action) == (1 - F.sigmoid(exit_action)).log()`.

Please review the changes for details.

### CODE REVIEW

**Comment by hyeok9855** [comment]
@younik, I don't understand. 
If we assign zero there, after softmax,

prob(AddEdge) = exp(0.0) / Z = 1.0 / Z
prob(Exit) = exp(exit_action) / Z
where Z = 1.0 + exp(exit_action).

This is not what we intended.

**Comment by younik** [comment]
I tested on my tutorial and it seems to lead to a more stable training. However, to see it, we need to update hyperparams; in my tutorial, I have to remove the scheduler, otherwise it was converging worse
